docs/releases/2.0.txt

```diff
@@ -163,7 +163,9 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* Commands that access migrations in a read-only manner, including
```

Comment:
I don't think this is a noteworthy feature that merits a mention in the release notes.